rpki-validator/src/main/java/net/ripe/rpki/validator3/storage/encoding/custom/RoaPrefixCoder.java

```diff
@@ -60,8 +67,11 @@ public RoaPrefix fromBytes(byte[] bytes) {
         long asn = Coders.toLong(content.get(ASN_TAG));
         byte[] maxLen = content.get(MAX_LEN_TAG);
         Integer maximumLength = maxLen != null ? Coders.toInt(maxLen) : null;
+        long notBefore = Coders.toLong(content.get(VALIDITY_BEFORE));
```

Comment:
How will this work when migrating from an old db? I think that `Coders.toLong` may throw an IllegalArgumentException.

Reply:
Added a guard checking availability of these fields. Now still testing old db/new version and how it behaves.

Reply:
Without DB clean up, these additional validity fields will not be updated, validator were not aware they are modified since hash is the same, so no refetch/update db. But it does not crash :)